examples/encfs/README.md

```diff
@@ -122,7 +133,7 @@ At this point, the `encfs-sidecar-args.json` file should be completely filled ou
 
 #### 7. Generate Security Policy
 
-At this point, the `aci-arm-template.json` file should be filled out except for the `ccepolicy` field. After installing the [Azure `confcom` CLI extension](#policy-generation), run the following command to generate the security policy and include the `--debug-mode` option so that the policy allows users to shell into the container. The --debug-mode option is only needed for testing purposes and should not be used in production, as it allows users to shell into the containers.
+Configure the [LogFile](aci-arm-template.json#L39) and [LogLevel](aci-arm-template.json#L43) fields on the `aci-arm-template.json`. eg. `log.txt` as the value of [LogFile] so that users can cat `log.txt` at the root of encrypted-filesystem-sidecar-container container. At this point, the `aci-arm-template.json` file should be filled out except for the `ccepolicy` field. Set the value of the `ccePolicy` to an empty string. After installing the [Azure `confcom` CLI extension](#policy-generation), run the following command to generate the security policy and include the `--debug-mode` option so that the policy allows users to shell into the container. The --debug-mode option is only needed for testing purposes and should not be used in production, as it allows users to shell into the containers.

```

Comment:
Note that the defaults are to print to stderr and at the warning level.

Reply:
How to use the default? Currently on our template, it is listed as the following: 
{
                  "name": "LogLevel",
                  "value": "<optional-loglevel-trace-debug-info-warning-error-fatal-panic>"
}
Would it use default if users leave it as an empty string?

Reply:
Yes, the default is used if it's an empty string https://github.com/microsoft/confidential-sidecar-containers/blob/2c76c71ab6610dbabb601ac0c1c9986df1e84151/cmd/skr/main.go#L41.